css/style.scss

```diff
@@ -605,13 +605,10 @@
 
       img {
         display: block;
-        max-height: 99%;
+        max-height: 100%;
         position: absolute;
         top: 0; bottom:0; left: 0; right:0;
         margin: auto;
-        //min-height: 100%;
-        //min-width: 100%;
-        //position: absolute;
```

Comment:
:laughing: Yeah, that's what I get for trying to push code with the kiddos around.